apps/demos/configs/React/config.js

```diff
@@ -125,6 +116,15 @@ window.config = {
 
     'devextreme': 'npm:devextreme/cjs',
     'devextreme-react': 'npm:devextreme-react/cjs',
+
+    'unified': 'bundles:unified/unified.bundle.js',
```

Comment:
wrap with identical begin/end comment (like in other files) and local this section at same place of this file everywhere